src/parity_testing/auth/testSuites/signOut/Test_that_global_signOut_error_returns_partial_success_with_global_sign_out_error.json

```diff
@@ -45,23 +46,17 @@
                 "revokeTokenError": {
                     "refreshToken": "eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJzdWIiOiJ1c2VySWQiLCJ1c2VybmFtZSI6InVzZXJuYW1lIiwiZXhwIjoxNTE2MjM5MDIyLCJvcmlnaW5fanRpIjoib3JpZ2luX2p0aSJ9.Xqa-vjJe5wwwsqeRAdHf8kTBn_rYSkDn2lB7xj9Z1xU",
                     "exception": {
+                        "errorType": "NotAuthorizedException",
+                        "errorMessage": "Failed to revoke token",
                         "recoverySuggestion": "See attached exception for more details. RevokeToken can be retried using the CognitoIdentityProviderClient accessible from the escape hatch.",
-                        "detailMessage": "Failed to revoke token",
                         "cause": {
-                            "detailMessage": "RevokeToken not attempted because GlobalSignOut failed.",
-                            "stackTrace": [
-                            ],
-                            "suppressedExceptions": [
-                            ]
-                        },
-                        "stackTrace": [
-                        ],
-                        "suppressedExceptions": [
-                        ]
+                            "errorType": "NotAuthorizedException",
+                            "errorMessage": null
+                        }
```

Comment:
The `json` structure didn't match the agreed design.

Reply:
@sktimalsina

Reply:
stacktrace, suppressedExceptions etc looks unnecessary. Perhaps the byproduct of custom serialization. It would make sense to keep as many items non-null as possible to validate parity properly.

Reply:
Agreed with the null messaging.. I can update that.